docs/source/faq.md

```diff
@@ -38,7 +38,7 @@ The UCX framework is maintained and supported by hardware vendors in addition to
 The framework design, data structures, and components are design to provide highly optimized access to the network hardware. 
 
 * **High level API for a broad range HPC programming models.**  
-UCX provides a high level API implemented in software 'UCP' to fill in the gaps across interconnects. This allows to use a single set of APIs in a library  to implement multiple interconnects. This reduces the level of complexities when implementing libraries such as Open MPI or OpenSHMEM.  Because of this, UCX performance portable  because a single implementation (in Open MPI or OpenSHMEM) will work efficiently on multiple interconnects. (e.g. uGNI, Verbs, libfabrics, etc). 
+UCX provides a high level API implemented in software 'UCP' to fill in the gaps across interconnects. This allows to use a single set of APIs in a library  to implement multiple interconnects. This reduces the level of complexities when implementing libraries such as Open MPI or OpenSHMEM.  Because of this, UCX performance portable  because a single implementation (in Open MPI or OpenSHMEM) will work efficiently on multiple interconnects. (e.g. uGNI, Verbs, libfabrics). 
```

Comment:
1. extra spaces (2 spaces before "Because" and "because"
2. Last sentence phrased weird.  IMO it should be
"Because of this, a single implementation in programming model libraries such as Open MPI or OpenSHMEM will work efficiently on multiple interconnects (uGNI, Verbs, TCP, shared memory, etc).

Reply:
Please remove libfabrics from the sentence.

Reply:
(uGNI, Verbs, TCP, shared memory, CUDA, ROCM, etc.)

Reply:
@yosefe what do you think:
```
UCX provides a high-level and performance-portable network API. The API implements API that targets a variety of programming models ranging from high-performance MPI implementation to Apache Spark. UCP API abstracts differences and fills in the gaps across interconnects implemented in the UCT layer. This allows using a UCP API  to implement support multiple interconnects. As a result, implementations of programming models and libraries (MPI, OpenSHMEM, Apache Spark, RAPIS, etc.) is simplified while providing efficient support for multiple interconnects (uGNI, Verbs, TCP, shared memory, ROCM, CUDA, etc.).
```

Reply:
"The API implements API that targets.." -> "The API targets.." or ", that targets.."
"This allows using a UCP API to implement support multiple interconnects" -> I'd remove this sentence; next sentence is basically a reiteration of it

Reply:
"UCX provides a high-level and performance-portable network API. The API targets a variety of programming models ranging from high-performance MPI implementation to Apache Spark. UCP API abstracts differences and fills in the gaps across interconnects implemented in the UCT layer. As a result, implementations of programming models and libraries (MPI, OpenSHMEM, Apache Spark, RAPIS, etc.) is simplified while providing efficient support for multiple interconnects (uGNI, Verbs, TCP, shared memory, ROCM, CUDA, etc.)."

Okay ?

Reply:
LGTM

Reply:
@changchengx FYI

Reply:
@shamisp Done
BTW, what's "MPI, OpenSHMEM, Apache Spark, ```RAPIS```"? Should it be "RAPIDS"?

Reply:
@changchengx , yes - rapids

Reply:
@shamisp Done